goblet/backends/backend.py

```diff
@@ -180,3 +182,6 @@ def _zip_directory(self):
 
     def get_environment_vars(self):
         raise NotImplementedError("get_environment_vars")
+
+    def check_or_enable_service(self, enable=False):
```

Comment:
might be confusing since we are importing a function with the same name. maybe change to `_check_or_enable_service` or import check_or_enable_service as `app_check_or_enable_service`